src/parser/warlock/demonology/modules/pets/PET_INFO.js

```diff
@@ -0,0 +1,81 @@
+const INNER_DEMON_NETHER_PORTAL_DURATION = 15000;
+
+const indexByGuid = obj => {
+  Object.keys(obj).forEach(key => {
+    const pet = obj[key];
+    obj[pet.guid] = pet;
```

Comment:
Not really, it should extend the object to be indexable by guids as well. SPELLS, ITEMS, SPECS and RESOURCE_TYPES are also indexed in the same way and it works fine (I just use `guid` property instead of `id`, that's why I made it myself instead of importing).